### PR TITLE
LPS-165830 - [Raylife AP] Creating additional fields when is auto quote

### PR DESCRIPTION
**Pull request by pedroliveirav**
[Task: LPS-165830](https://issues.liferay.com/browse/LPS-165830)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx [hunk 1]
```diff
@@ -82,6 +113,15 @@ const QuotedSummary = ({externalReferenceCode}: externalReferenceCodeType) => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	function handleValuesFormatter(value: string) {
+		if (value === bodilyInjury || motoristBodilyInjury) {
+			return value.replaceAll(',000', ',000.00');
+		}
+		else {
+			return value.replace(',000', ',000.00');
+		}
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx:122]: Please, remove the ```else``` condition.

```suggestion
		if (value === bodilyInjury || motoristBodilyInjury) {
			return value.replaceAll(',000', ',000.00');
		}
		
		return value.replace(',000', ',000.00');
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions

**Comment by eliasantosf** [comment]
Just starting reviewing :)

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx (hunk 1 above)
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx]: Done.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx [hunk 2]
```diff
@@ -28,18 +29,48 @@ const QuotedSummary = ({externalReferenceCode}: externalReferenceCodeType) => {
...
+
+		setBodilyInjury(dataJSON?.coverage?.form?.bodilyInjury);
+		setMedicalPayments(dataJSON?.coverage?.form?.medical);
+		setPropertyDamage(dataJSON?.coverage?.form?.propertyDamage);
+		setMotoristBodilyInjury(
+			dataJSON?.coverage?.form?.uninsuredOrUnderinsuredMBI
+		);
+		setMotoristPropertyDamage(
+			dataJSON?.coverage?.form?.uninsuredOrUnderinsuredMPD
+		);
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx]: Done.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx [hunk 3]
```diff
@@ -28,18 +29,48 @@ const QuotedSummary = ({externalReferenceCode}: externalReferenceCodeType) => {
...
 
-		const quoteRequest = await getQuotes();
+		const quoteRequest = await getQuotes(PARAMETERS_GET_ALL_ITEMS);
 		const quoteItems = quoteRequest?.data?.items;
 		const quoteItem = quoteItems.find(
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx]: Done.

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx [hunk 4]
```diff
@@ -48,7 +79,7 @@ const QuotedSummary = ({externalReferenceCode}: externalReferenceCodeType) => {
 		);
 		const quoteId = quoteItem.id;
 
-		const policiesRequest = await getPolicies();
+		const policiesRequest = await getPolicies(PARAMETERS_GET_ALL_ITEMS);
 		const policiesItems = policiesRequest?.data?.items;
 		const policyItem = policiesItems.find(
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/quoted-summary/index.tsx]: Done.

**Review by joanagrego — APPROVED** [review]
Approved! Thanks @pedroliveirav :white_check_mark:

**Review by silasbsi — APPROVED** [review]

**Comment by silasbsi** [comment]
ci:forward

**Comment by silasbsi** [comment]
ci:test:stop

**Comment by silasbsi** [comment]
ci:stop

**Comment by pedroliveirav** [comment]
ci:test:stop

**Comment by pedroliveirav** [comment]
ci:stop

**Comment by pedroliveirav** [comment]
ci:close

**Pull request closed without merging**